src/main/resources/flattening/config/bulk/env.conf

```diff
@@ -0,0 +1,70 @@
+base_path = "/shared/Observapur/staging/Flattening"
+
+schema_file_path = [
+  "schema/bulk/DCIR_schema.csv"
+  "schema/bulk/MCO_schema.csv"
+  "schema/bulk/SSR_schema.csv"
+  "schema/bulk/HAD_schema.csv"
+]
+
+tables_config = [
+  {include "dcir.conf"}
+  {include "mco.conf"}
+  {include "ssr.conf"}
+  {include "had.conf"}
+]
+
+join = [
+  {
+    name = "DCIR"
+    join_keys = [
+      "DCT_ORD_NUM"
+      "FLX_DIS_DTD"
+      "FLX_EMT_NUM"
+      "FLX_EMT_ORD"
+      "FLX_EMT_TYP"
+      "FLX_TRT_DTD"
+      "ORG_CLE_NUM"
+      "PRS_ORD_NUM"
+      "REM_TYP_AFF"
+    ]
+    main_table_name = "ER_PRS_F"
+    tables_to_join = ["ER_PHA_F","ER_CAM_F", "ER_ETE_F"]
+  }
+  {
+    name = "MCO"
+    join_keys = ["ETA_NUM", "RSA_NUM"]
+    main_table_name = "MCO_B"
+    pmsi_patient_table_name = "MCO_C"
+    tables_to_join = ["MCO_A", "MCO_D", "MCO_UM"]
+  }
+  {
+    name = "MCO_CE"
+    join_keys = ["ETA_NUM", "SEQ_NUM"]
+    main_table_name = "MCO_FASTC"
+    pmsi_patient_table_name = "MCO_CSTC"
+    tables_to_join = ["MCO_FMSTC", "MCO_FBSTC"]
```

Comment:
You need to add a new table "MCO_FCSTC".

Reply:
OK